ui/src/app/applications/components/application-details/application-resource-filter.tsx

```diff
@@ -105,15 +107,38 @@ export const Filters = (props: {
         return groupedFilters[prefix] ? groupedFilters[prefix].split(',').map(removePrefix(prefix)) : [];
     };
 
```

Comment:
I wonder if we need to show the number of items hidden in addition to the number of items currently visible in the view.  eg. that way, we could show how many of the items will be shown once a filter is removed (Josh's issue)

Reply:
I agree that Josh has asked for hinting at how many resources will be visible once each filter is removed, but I didn't find a good place for it, (tooltip for each filter???, it doesn't make sense near the CLEAR button because that needs to represent all the types of the specific filters egs- health, sync - which again isn't specific). I implemented it like this because it addresses the main problem i.e. confusion as to how what filter click renders what result, thus clicking a field that mentions 0 count would obviously result in an empty tree. Still open to hear more ideas.

Reply:
Hey @saumeya , yeah not sure what the UI design will be ideal for this.  Maybe add it to the filter group?

Reply:
Agreed Keith, I have another idea to implement `hints`, I will try to address that in another PR based on whether this gets approved. I have removed the closes issue from description for now and this Pr is solely for counts.